packages/rspack/src/util/index.ts

```diff
@@ -115,7 +118,8 @@ const getDeprecationStatus = () => {
 	const defaultEnableDeprecatedWarning = true;
 	if (
 		process.env.RSPACK_DEP_WARNINGS === "false" ||
-		process.env.RSPACK_DEP_WARNINGS === "0"
+		process.env.RSPACK_DEP_WARNINGS === "0" ||
+		isCI
```

Comment:
I don't think we should disable deprecation warnings for every user. Only warnings in Rspack repo should be disabled instead.

Reply:
i will fix it